src/test/text-fragment-utils-test.js

```diff
@@ -32,7 +32,8 @@ describe('TextFragmentUtils', function () {
     );
   });
 
-  it('works with complex layouts', function () {
+  // This test is disabled until textEnd support is back in place.
```

Comment:
nit: TODO ?

Reply:
The relevant TODO is in the util file :)